downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/MolecularFunctionAnnotationBuilder.java

```diff
@@ -0,0 +1,116 @@
+package org.reactome.release.downloadDirectory.GenerateGOAnnotationFile;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.schema.SchemaClass;
+
+import java.util.*;
+
+public class MolecularFunctionAnnotationBuilder {
+
+    private static final List<String> microbialSpeciesToExclude = new ArrayList<>(Arrays.asList("813", "562", "491", "90371", "1280", "5811"));
```

Comment:
No need to include `new ArrayList<>(...)` here.  Using `Arrays.asList(...)` will return an ArrayList object.  The same is also true for the other Builder classes.